internal/monitors/vsphere/metadata.yaml

```diff
@@ -0,0 +1,1093 @@
+monitors:
+  - monitorType: vsphere
+    doc: |
+      A VMware vSphere deployment includes physical hosts, ESXi hypervisors, virtual machines, and vCenter Server.
+      This monitor reports metrics for a vSphere deployment by logging into a vCenter Server and periodically
+      retrieving data about the deployment and its real time performance data.
+
+      When this monitor starts up, it logs into vCenter Server and traverses the inventory, gathering and caching all of
+      the hosts and virtual machines, and their available metrics.
+
+      Then it periodically queries the vCenter for the performance data for those metrics. It does so at a rate of
+      once every 20 seconds, the interval at which vCenter makes real-time performance data avaialable.
```

Comment:
So if I set `intervalSeconds: 10` on the montior what does that do?  That will be the default btw.

Reply:
Any value should be fine (I did test 10s). If it's too soon to request an interval, the number of intervals requested will be zero and no query will occur. If a lot of time has passed, as many intervals required to fill in the gap will be requested.